### DISCUSSION TITLE
Signing commits

### DISCUSSION THREAD

**Discussion by satanTime** [renovatebot/renovate#8019] [Request Help] [answered]
I use your bot for GitHub.

**Describe the bug**
With my work everything works fine, but when I created a PR against the main repo I got the error:
**Merging is blocked. The base branch requires all commits to be signed.**
https://github.com/ike18t/ng-mocks/pull/260

Here is the config: https://github.com/satanTime/ng-mocks/blob/master/renovate.json

**Additional info**
Looks like it should be have been solved long time ago by #897, but I found out that it doesn't work.

**Expected behavior**
Would be great if the bot signed commits.
Currently we plan to disable this check in the main repo.

**Comment by rarkins** [comment]
Use the offical app instead and it will sign commits: https://github.com/apps/renovate

Or else you need to configure your own git private key in your bot config.

**Reply by satanTime** [reply]
Does the github bot uses an unofficial app?

When I use your link it says:
Renovate: Installed 16 days ago

That's the time when I started out using the bot.

**Reply by satanTime** [reply]
Ah... looks like rebase is the problem... My bad.

**Comment by satanTime** [comment, marked as answer]
The problem with a missed signature is caused by `rebase` merge strategy. If you need a commit being signed, please use normal `merge`, not `rebase` or `squash`.
